dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -760,6 +760,10 @@ error.checkpoint.failed=CWWKE0962E: The server checkpoint request failed. The fo
 error.checkpoint.failed.explanation=A checkpoint request for the server failed. No valid checkpoint process image for the server was created.
 error.checkpoint.failed.useraction=To determine why the checkpoint failed, inspect the log files in the checkpoint directory that is located in the server logs directory.
 
+error.checkpoint.failed.selinux.netdiag=CWWKE0963E: The server checkpoint request failed because netlink system calls may be blocked. If SELinux is enabled and enforcing then netlink system calls can be blocked. Either disable SELinux or enable the netlink system calls with the command "setsebool virt_sandbox_use_netlink 1".
```

Comment:
```suggestion
error.checkpoint.failed.selinux.netdiag=CWWKE0963E: The server checkpoint request failed because netlink system calls might be blocked. If SELinux is enabled and enforcing then netlink system calls can be blocked. Either disable SELinux or enable the netlink system calls with the  "setsebool virt_sandbox_use_netlink 1" command.
```

> If SELinux is enabled and enforcing ..

I had to read this twice to understand, it might be more clear if we we can say what is being enforced, otherwise users/translators might get confused.

eg If SELinux is enabled and enforcing <permissions>...  
Or whatever rule/restriction is being enforced

Reply:
I hadn't thought of that but I think it would help. My comment was more about "enforcing" -  what is being enforced?

Reply:
OK
SELinux can run in one of three modes: `disabled`, `permissive`, or `enforcing`.

Using the `disabled` mode means that no rules from the SELinux policy are applied and the system is not protected.
In the `permissive` mode, SELinux is active, the security policy is loaded, the file system is labeled and access denial entries are logged. However, the policy is not enforced and thus no access is actually denied.

In the `enforcing` mode, the security policy is applied. Each access that is not explicitly allowed by the policy is denied.  With that in mind, how does the following read?

> error.checkpoint.failed.selinux.netdiag=CWWKE0963E: If SELinux is enabled and using enforcing mode then netlink system calls can be blocked. The server checkpoint request failed because netlink system calls might be blocked by the SELinux "virt_sandbox_use_netlink" policy setting. Either disable SELinux or enable the netlink system calls with the "setsebool virt_sandbox_use_netlink 1" command.

Reply:
ok makes sense now. I think the update looks good but looking again I would replace "can" with "might" in the first sentence for consistency

> If SELinux is enabled and using enforcing mode then netlink system calls might be blocked.

Reply:
actually- lets make sure I understand properly. 

> The server checkpoint request failed because netlink system calls might be blocked...

is the intended meaning:

1. the checkpoint fails automatically simply because the possibility exists that netlink calls can be blocked
2. the checkpoint failed and the most like cause is that netlink calls were actually blocked

Reply:
More close to meaning number 2.  We know the checkpoint failed because criu could not successfully make system calls to netlink.  One possible cause (which we think is very likely) is that SELinux is blocking the system call.

Reply:
ok- in that case how about

> error.checkpoint.failed.selinux.netdiag=CWWKE0963E: The server checkpoint request failed because netlink system calls were unsuccessful. If SELinux is enabled in enforcing mode then netlink system calls might be blocked by the SELinux "virt_sandbox_use_netlink" policy setting. Either disable SELinux or enable the netlink system calls with the "setsebool virt_sandbox_use_netlink 1" command.